chore(plugin-template): add missing semicolon (#24638)
backstage/backstage: .changeset/tasty-moles-jog.md

---
'@backstage/cli': patch
---

> Add previously-missing semicolon to tsx file templated by `backstage-cli new --select plugin`.

Reviewer: ```suggestion
Add previously-missing semicolon in file templated by `backstage-cli new --select plugin`.
```
